examples/adaptivity.py

```diff
@@ -1,41 +1,40 @@
-# Adaptive refinement of reentrant corner
-#
-# In this script we solve the Laplace problem on a unit square that has the
-# bottom-right quadrant removed (a.k.a. an L-shaped domain) with Dirichlet
-# boundary conditions matching the harmonic function
-#
-#     ³√(x² + y²) cos(⅔ arctan2(y+x, y-x))
-#
-# shifted by ½ such that the origin coincides with the middle of the unit
-# square. Note that the function evaluates to zero over the two boundaries
-# bordering the removed quadrant.
-#
-# This benchmark problem is known to converge suboptimally under uniform
-# refinement due to a singular gradient in the reentrant corner. This script
-# demonstrates that optimal convergence rates of -(p+1)/2 for the L2 norm and
-# -p/2 for the H1 norm can be restored by using adaptive refinement.
-
-from nutils import mesh, function, solver, export, cli, testing
+from nutils import mesh, function, solver, export, testing
 from nutils.expression_v2 import Namespace
 import numpy
 import treelog
 
 
-def main(etype: str, btype: str, degree: int, nrefine: int):
-    '''
-    Adaptively refined Laplace problem on an L-shaped domain.
-
-    .. arguments::
-
-       etype [square]
-         Type of elements (square/triangle/mixed).
-       btype [h-std]
-         Type of basis function (h/th-std/spline), with availability depending on
-         the configured element type.
-       degree [2]
-         Polynomial degree
-       nrefine [5]
-         Number of refinement steps to perform.
+def main(etype: str = 'square',
+         btype: str = 'h-std',
+         degree: int = 2,
+         nrefine: int = 5):
+
+    '''Adaptively refined Laplace problem on an L-shaped domain.
+    
+    Solves the Laplace problem on a unit square that has the bottom-right
+    quadrant removed (a.k.a. an L-shaped domain) with Dirichlet boundary
+    conditions matching the harmonic function
+    
+        ³√(x² + y²) cos(⅔ arctan2(y+x, y-x))
+    
+    shifted by ½ such that the origin coincides with the middle of the unit
+    square. Note that the function evaluates to zero over the two boundaries
+    bordering the removed quadrant.
```

Comment:
'touching' instead of 'bordering'?

Reply:
changed to "where the domain borders the removed quadrant"